### ISSUE TITLE
Should be able to set "real path" on a whole folder and subfolders

### ISSUE THREAD

**Issue by jon64digital** [storyblok/storyblok#506]
**The feature would affect:** (check one with "x")   
- [ ] *app.storyblok.com (CMS - Interface)*   
- [ ] *api.storyblok.com (CMS - Content Delivery API)*
- [ ] *mapi.storyblok.com (CMS - Management API)*
- [ ] *capi.storyblok.com (Commerce - API)*     
- [ ] *Commerce - Interface*   
- [ ] *Other* 

**Is your feature request related to a problem? Please describe.**

The ability to put in the "real path" is useful but at the moment it has to be done on each new story and this means we have to rely on our client or content editor to know how to do this.

**Describe the solution you'd like**

The following scenarios would be useful to solve:

1) If you have a folder with the slug "home/products/subfolder/product-name" you might want to set it so that all products have the real path "/products/product-name".

2) If you have a folder you want to ignore, let's say any slug which looks like "home/data/.../.../story-name" needs to have to /data folder removed from the real path

You could currently do both of these on a story by story basis but there's nowhere to add a rule for all items in a folder.

**Comment by bstrdChronicles** [comment]
I totally agree - especially for handling use cases like product previews it would make a lot of sense to have the possible on folder level.

In addition to that it would make a lot of sense to make the real path also possible as a selection of a route like in the route selector of a link field. ( in the story itself and also on folder level )

That would not only reduce a lot of possible problems with hardcoding routes in there, but also adapt if a route / slug of a story then changes.

**Comment by fcisio** [comment]
What's the status of this in 2021?
This is something very interesting that I would love to see implemented.

**Comment by Joobs** [comment]
Has any progress been made of this?

**Comment by alvarosabu** [comment]
Hi @JonCognioDigital, we are closing this issue because this repository was used in the early days for support-related questions, but we've since transitioned to more effective channels for helping our users.

Please, refer to the following resources:

Storyblok Github Org: https://github.com/storyblok
Storyblok Docs: https://www.storyblok.com/docs
Official Support: https://support.storyblok.com
Join our Discord: https://www.storyblok.com/join-discord

Thanks for the understanding 🙏🏻
